packages/react-native-client/src/utils/errorListener.ts

```diff
@@ -0,0 +1,13 @@
+import { ReceivableEvents } from '../hooks/useFishjamEvent';
+import { nativeModuleEventEmitter } from '../RNFishjamClientModule';
+
+export const initializeWarningListener = () => {
+  if (__DEV__) {
+    nativeModuleEventEmitter.addListener(
```

Comment:
Just in case, can you wrap this code into some try/catch? Just to avoid app crashing on start just by importing our lib